include/boost/math/policies/error_handling.hpp

```diff
@@ -348,32 +376,36 @@ inline constexpr T raise_overflow_error(
 }
 
 template <class T>
-inline T raise_overflow_error(
+BOOST_MATH_GPU_ENABLED inline T raise_overflow_error(
            const char* ,
            const char* ,
            const  ::boost::math::policies::overflow_error< ::boost::math::policies::errno_on_error>&) BOOST_MATH_NOEXCEPT(T)
 {
+   #ifndef BOOST_MATH_SYCL_ENABLED
```

Comment:
I was thinking adding a GPU default policy of:

```
using gpu_policy = policy<
      promote_float<false>,
      promote_double<false>,
      overflow_error<ignore_error>
      >;
```

I guess this could also just overwrite the value of `policies::policy<>()` when `BOOST_MATH_HAS_GPU_SUPPORT` is defined?

Reply:
Right, when building for the device, set the default error handling to ignore_error for everything, ie:

```
#    define BOOST_MATH_DOMAIN_ERROR_POLICY ignore_error
#     define BOOST_MATH_POLE_ERROR_POLICY ignore_error
#     define BOOST_MATH_OVERFLOW_ERROR_POLICY ignore_error
#     define BOOST_MATH_EVALUATION_ERROR_POLICY ignore_error
#     define BOOST_MATH_ROUNDING_ERROR_POLICY ignore_error
```

Currently the default when exceptions are disabled is `errno_on_error`, but we have no access to `::errno` so don't want that here.